Coverity scan fails because of  include for 90%

I try to compile xournalpp with cov-build for coverity scan. But it is failing on same extern error in `/usr/include/c++/8/type_traits` for almost all files. This results in 90% failure. Anybody an idea?

```
"/usr/include/c++/8/type_traits", line 1049: error #255: type name is not
          allowed
        : public __bool_constant<__is_assignable(_Tp, _Up)>
                                                 ^

"/usr/include/c++/8/type_traits", line 1049: error #255: type name is not
          allowed
        : public __bool_constant<__is_assignable(_Tp, _Up)>
                                                      ^

"/usr/include/c++/8/type_traits", line 1049: error #20: identifier
          "__is_assignable" is undefined
        : public __bool_constant<__is_assignable(_Tp, _Up)>
                                 ^

"/usr/include/c++/8/utility", line 307: error #1937: pack expansion does not
          make use of any argument packs
        using __type = _Index_tuple<__integer_pack(_Num)...>;
                                                        ^

Emit for file 'PROJECTPATH/FILEPATH/FILE.cpp' complete.
[ERROR] 4 errors detected in the compilation of 
```

--- a/README.md
+++ b/README.md
@@ -2,11 +2,6 @@
 
 [![Build Status](https://travis-ci.org/xournalpp/xournalpp.svg?branch=string_new)](https://travis-ci.org/xournalpp/xournalpp)
 [![Join the chat at https://gitter.im/xournalpp/xournalpp](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/xournalpp/xournalpp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-<a href="https://scan.coverity.com/projects/xournal">
-  <img alt="Coverity Scan Build Status"
-       src="https://scan.coverity.com/projects/17046/badge.svg"/>
-</a>
-
 
 ## Features
 

--- a/check-source.sh
+++ b/check-source.sh
@@ -1,0 +1,4 @@
+#!/bin/bash
+
+cppcheck --quiet ./src -isrc/mathtex/
+

--- a/src/control/xml/XmlImageNode.h
+++ b/src/control/xml/XmlImageNode.h
@@ -33,5 +33,5 @@ private:
 
 	OutputStream* out;
 	int pos;
-	unsigned char buffer[30];
+	unsigned char buffer[30] = { 0 };
 };

--- a/src/control/xml/XmlTexNode.h
+++ b/src/control/xml/XmlTexNode.h
@@ -33,5 +33,5 @@ private:
 
 	OutputStream* out;
 	int pos;
-	unsigned char buffer[30];
+	unsigned char buffer[30] = { 0 };
 };

--- a/src/view/background/BaseBackgroundPainter.h
+++ b/src/view/background/BaseBackgroundPainter.h
@@ -46,7 +46,7 @@ private:
 	XOJ_TYPE_ATTRIB;
 
 protected:
-	BackgroundConfig* config;
+	BackgroundConfig* config = NULL;
 	PageRef page;
 	cairo_t* cr;
 

